cpp/ps0001/ps0001_Solution.cpp

```diff
@@ -1,31 +1,19 @@
-/* help your brother solve summation problems
-    155
-    940
-   +168
-  --------
-   1263
-*/
-
-
 #include<iostream>
 
 int main() {
+    // prompting user for 3 numbers to add 1/2
+    short number1;
+    short number2;
+    short number3;
 
-    int number1;
-    int number2;
-    int number3;
-
-    
-   
     std::cin >> number1;
-    std::cin  >> number2;
-    std::cin  >> number3;
-
+    std::cin >> number2;
+    std::cin >> number3;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add